.github/merge_rules.yaml

```diff
@@ -395,3 +395,5 @@
   - EasyCLA
   - Lint
   - pull
+
+- flaky_rules_location_url: https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/flaky-rules.json
```

Comment:
We should try not to add errant configuration values that don't match `MergeRules` to this since internal `github-export-checks` uses this file and is typically behind what is on Github master.